sample/src/main/java/co/lujun/sample/MainActivity.java

```diff
@@ -132,6 +139,19 @@ public void onTagCrossClick(int position) {
 
         // After you set your own attributes for TagView, then set tag(s) or add tag(s)
         mTagContainerLayout1.setTags(list1);
+        for (int i=0; i<list1.size(); i++) {
+            final int index = i;
+            Glide.with(mTagContainerLayout1.getContext())
+                    .asBitmap()
+                    .load("https://d1marr3m5x4iac.cloudfront.net/images/block/movies/17214/17214_aa.jpg")
+                    .into(new SimpleTarget<Bitmap>() {
+                        @Override
+                        public void onResourceReady(Bitmap resource, Transition<? super Bitmap> transition) {
+                            mTagContainerLayout1.getTagView(index).setImage(resource);
```

Comment:
I think you might want to update the code to:
- move the `for` loop within this block

or

- add a `sleep` after each iteration

since as it is right now, I'm afraid can be inclined to some concurrency issues

Reply:
Leaving `for` loop as is, as I meant to showcase various images - I've updated the code to do this.

I added a `sleep` in, and refactored this code a bit (because that function was getting giant). 

Can you explain a bit more about why you're concerned about concurrency here?

Reply:
I guess I was just worried about the fact that we're asking Glide to fetch the same assets in a `for` loop. For this demo purpose I really like your updated solution, 👍